sonar-db/src/main/java/org/sonar/db/issue/IssueFilterDao.java

```diff
@@ -45,15 +46,23 @@ public IssueFilterDto selectById(long id) {
     }
   }
 
+  /**
+   * @deprecated since 5.2 use {@link #selectByUser(DbSession, String)}
+   */
+  @Deprecated
   public List<IssueFilterDto> selectByUser(String user) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)